src/plugins/legacy/polygonRoi/polygonEventFilter.cpp

```diff
@@ -203,15 +257,14 @@ bool polygonEventFilter::leftButtonBehaviour(medAbstractView *view, QMouseEvent
     switch (cursorState)
     {
     case CURSORSTATE::CS_MOUSE_EVENT:
-    case CURSORSTATE::CS_SLICE_CHANGED:
     {
         return addPointInContourWithLabel(mouseEvent);
     }
     case CURSORSTATE::CS_CONTINUE:
     {
         for (medTagRoiManager *manager : managers)
         {
-            manager->setContourEnabled(true);
+            //manager->setContourEnabled(true);
```

Comment:
you can remove

Reply:
true

Reply:
done

Reply:
the PR is already merged, and it's not vital, but the line is still here